src/types/exchange.ts

```diff
@@ -0,0 +1,6 @@
+export interface Exchange {
+  providerAddr: string
+  id: string
+  //   assets: (StableToken | CollateralAsset)[]
```

Comment:
No, it will work but would require additional refactoring. I decided to defer that until the exchange service gets added. I will remove the comment for now, but this will come back in the next PR.